docs/troubleshooting.md

```diff
@@ -65,24 +65,10 @@ If your topology allow it, you may configure Contour to catch `nginx` ingress cl
 
 ## I've deployed on Minikube and nothing seems to work
 
-Minikube remaps Contour's service loadbalancer from ports 80 and 443 to a random high port as the ingress.
-This is problematic because this port is not a _well known_ (see RFC 2616) port so `curl(1)` or browsers will include the port number in the `Host:` header.
-This causes Envoy to misroute the request because the domain name in the RDS virtualhost entry does not contain the `:port` suffix assigned by Minikube.
-
-The problem is the port minikube chooses is not easily predictable, so it is not simply a matter of including various permutations of hostname:port in the virtualhost.domains array.
-
-### Workarounds
-The workaround if you are using minikube is to either force the `Host:` header with something like
-
-    curl -H "Host: example.com" -v http://example.com:31847
-
-Or use a OSX specific extension to curl
-
-    curl -v --connect-to example.com:80:example.com:31847 http://example.com/
-
-This is tracked as issue [#210][1], which is blocked on [envoyproxy/envoy#1269][2].
-At the moment there is no ETA when these issues will be resolved.
+Minikube is not recommended as a platform to test or develop Contour due to its network limitations.
+A [seperate document][3] details deployment and troubleshooting information when working with Minikube.
 
```

Comment:
"Minikube is not recommended for testing or developing Contour because of its network limitations. To run Contour on Minikube for exploration only, see [Contour on Minikube][3]